app/src/main/java/com/example/calculadoramvp/mvp/model/CalculatorModel.java

```diff
@@ -2,7 +2,54 @@
 
 public class CalculatorModel {
 
-    private int operando_1;
-    private String operador;
-    private float resultado;
+    private String operand_1;
+    private String operand_2;
+    private String operator;
+    private float result;
+
+    public void cleanVisor() {
+        operand_1 = "";
```

Comment:
Please, remove the hardcode into a StringUtil.

Reply:
done